lib/import_export/client.rb

```diff
@@ -20,6 +20,11 @@ def search(params = {})
       parse_response Query.new(params, api_key).call
     end
 
+    def overall_total(params = {})
+      response = Query.new(params, api_key).call
+      JSON.parse(response)['total']
+    end
+
     private
 
     def parse_response(response)
```

Comment:
Perhaps we could create a small array like class that could be extended to add on the total_results:

```ruby
class ResultsArray < Array
  attr_acessor :total_results
end

###

def parse_response(response)
  payload = JSON.parse(response)
  ResultsArray.new(payload['results'].map { |data| Result.new(data) }).tap do |results|
    results.total_results = payload['total']
  end
end
```